include/mesos_state.hrl

```diff
@@ -20,7 +20,7 @@
 -type role() :: binary().
 -type slave_id() :: binary().
 -type hostname() :: binary().
--type protocol() :: tcp | udp.
+-type protocol() :: tcp | udp | client.
```

Comment:
I don't think the user intentionally created the new type, tracing where the "client" came from will take some digging. We have the marathon json to replicate the issue so I think I should probably start there.